app/src/main/res/values/styles.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
```

Comment:
Nice!
I suggest adding all the text styles now, so you don't have to worry about them in the future